autorest/codegen/__init__.py

```diff
@@ -340,11 +340,6 @@ def get_yaml(self) -> Dict[str, Any]:
             input_yaml = self._autorestapi.get_value("input-yaml")
             file_content = self._autorestapi.read_file(input_yaml)
         else:
-            inputs = self._autorestapi.list_inputs()
-            _LOGGER.debug("Possible Inputs: %s", inputs)
-            if "code-model-v4-no-tags.yaml" not in inputs:
-                raise ValueError("code-model-v4-no-tags.yaml must be a possible input")
-
```

Comment:
the check logic is duplicated with :
https://github.com/Azure/autorest.python/blob/90868608454c56c60d6cfa0f0bd769df9fcf8b74/autorest/codegen/__init__.py#L334-L337